stories/index.stories.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+import { withInfo } from '@storybook/addon-info';
+
+import '../less/styles.less';
+
+
+import {Button, ButtonGroup, Breadcrumb} from 'react-bootstrap';
+
+//Stories of components
+
+storiesOf('Breadcrumb', module)
+  
+  .add('Default', 
+  	withInfo('') 
+  		(
+		() => (<div>
+					<Breadcrumb>
```

Comment:
This'd be a good candidate file to try running eslint on, now that you've copied over an eslint config. I think `./node_modules/.bin/eslint --fix stories/index.stories.js` will fix a lot of stuff automatically.